src/utils/constants.js

```diff
@@ -4,8 +4,10 @@ export const SYSTEM_FONT =
   '-apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Helvetica Neue", Helvetica, sans-serif';
 
 export const KEYBOARD_SHORTCUTS = {
-  PRESENTER_MODE: 'mod+p',
-  OVERVIEW_MODE: 'mod+o',
+  PRESENTER_MODE: 'mod+shift+p',
+  OVERVIEW_MODE: 'mod+shift+o',
+  PRINT_MODE: 'mod+shift+r',
+  EXPORT_MODE: 'mod+shift+e',
```

Comment:
I'm not sold on these but it's best to make most include <kbd>shift</kbd> since many browsers do things on just <kbd>meta</kbd>.